react/week1/todolist/src/App.js

```diff
@@ -0,0 +1,64 @@
+import './App.css';
+
+const myTodo = [
+  {
+    id: 1,
+    task: "Finish my hyf homework",
+    dueDate: "Tue 18-05-2021"
+  },
+  {
+    id: 2,
+    task: "Shop groceries",
+    dueDate: "Wed 19-05-2021"
+  },
+  {
+    id: 3,
+    task: "Do laundry",
+    dueDate: "Thu 20-05-2021"
+  }
+]
+
+function App() {
+  return (
+    <div className="App">
+      <h1>Todo List</h1>
+      <div className="container">
+        <Table />
+      </div>
+    </div>
+  );
+}
+
+function Table() {
+  return (
+    <table className="todo-table">
+      <tr>
```

Comment:
Chech the dev tools. You get an error regarding the table. You can fix it by adding tbody